### PR TITLE
#15391: Add section on service-version header to interface-doc

### PR DESCRIPTION
**Pull request by SzilardBaizat**
In this workitem we added a new section into the general operation-modes markdown file, where you can found out how to access a specific signaturcloud version.

### CODE REVIEW

File: doc/general/operation-modes/operation-modes.md [hunk 1]
```diff
@@ -34,6 +34,38 @@ The queue nutshell serves to encapsulate the functionality of a receipt chain fo
 
 The Helper nutshell is used to support updates and configurations.
 
+#### fiskaltrust.SignatureCloud
+
+The fiskaltrust.SignatureCloud is a pure online solution. The receipt linking is handled entirely online. The advantage of this solution is that no installation or configuration is required for the client and any platform can use this service.
+
+The fiskaltrust.SignatureCloud has two different versions:
```
Comment (StefanKert) [inline doc/general/operation-modes/operation-modes.md]: Should we add information on the support policy here? That we support the current version (default version if you don´t use a version header) and the last minor version. @steininger

File: doc/general/operation-modes/operation-modes.md [hunk 2]
```diff
@@ -34,6 +34,38 @@ The queue nutshell serves to encapsulate the functionality of a receipt chain fo
...
+ - 1.1
+ - 1.2
+
+We differentiate between them by backend servers. So the external url is the same: https://signaturcloud.fiskaltrust.at for both versions. If you want to use a specific version of fiskaltrust.SignatureCloud, you will need to set the service-version parameter into the header. As default the fiskaltrust.SignatureCloud 1.1 version is used.
```
Comment (StefanKert) [inline doc/general/operation-modes/operation-modes.md]: Maybe we should be more specific about http headers here. IMO it makes sense to reference a wiki article or an general information on http headers here.

Comment (steininger) [inline doc/general/operation-modes/operation-modes.md]: in future updates we need to clarify market-related top-level-domain.

Comment (steininger) [reply doc/general/operation-modes/operation-modes.md]: in future version we need to clarify this.

Comment (SzilardBaizat) [reply doc/general/operation-modes/operation-modes.md]: is it ok for you if i add the following link (https://en.wikipedia.org/wiki/List_of_HTTP_header_fields) as a reference where you can find more information about the http headers? or i can write a short description about it, as you wish

Comment (SzilardBaizat) [reply doc/general/operation-modes/operation-modes.md]: what do you think moving this section into appendix-at-rksv/operation-modes/operation-modes.md file can be a solution for this market-related top-level-domain issue?

File: doc/general/operation-modes/operation-modes.md (hunk 1 above)
Comment (StefanKert) [reply doc/general/operation-modes/operation-modes.md]: We will keep it as it is for now and adapt the interface-doc if necessary.

File: doc/general/operation-modes/operation-modes.md [hunk 3]
```diff
@@ -34,6 +34,38 @@ The queue nutshell serves to encapsulate the functionality of a receipt chain fo
 
 The Helper nutshell is used to support updates and configurations.
 
+#### fiskaltrust.SignatureCloud
```
Comment (TSchmiedlechner) [inline doc/general/operation-modes/operation-modes.md]: SignaturCloud is written without an "e", also applies for below.

File: doc/general/operation-modes/operation-modes.md (hunk 1 above)
Comment (TSchmiedlechner) [inline doc/general/operation-modes/operation-modes.md]: Maybe add "currently has two..", as this is likely to change in the future.

File: doc/general/operation-modes/operation-modes.md (hunk 3 above)
Comment (SzilardBaizat) [reply doc/general/operation-modes/operation-modes.md]: corrected

File: doc/general/operation-modes/operation-modes.md (hunk 1 above)
Comment (SzilardBaizat) [reply doc/general/operation-modes/operation-modes.md]: added

**Pull request merged**